debezium-core/src/main/java/io/debezium/relational/ColumnEditor.java

```diff
@@ -39,6 +39,14 @@
     int jdbcType();
 
     /**
+     * Get the underlying type for this column where
+     * jdbcType() itself is the generic Types.ARRAY.
+     *
```

Comment:
Does this return the same thing as `jdbcType()` if the type is _not_ an array? Documenting the behavior would be helpful in the future.